src/main/java/com/smoothstack/utopia/counter/service/FlightService.java

```diff
@@ -33,9 +33,7 @@
 	
 	public Flight readFlight(Integer flightId) throws InvalidIdException {
 		Optional<Flight> flight = flightRepo.findById(flightId);
-		if (!flight.isPresent()) {
-			throw new InvalidIdException("That ID is invalid.");
-		}
+		flight.orElseThrow(() -> new InvalidIdException("Invalid Airport Code."));
```

Comment:
i think you can chain orElseThrow directly to return object without using get and will return object inside optional